addons/l10n_be_edi/models/mail_template.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models
+
+from lxml import etree
+import base64
+from xml.sax.saxutils import escape, quoteattr
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    def _l10n_be_edi_inject_pdf_into_xml(self, pdf_name, pdf, xml):
+        '''Adds the pdf report in the e-fff XML as base64 string.
+        '''
+        to_inject = '''
+            <cac:AdditionalDocumentReference xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2">
+                <cbc:ID>%s</cbc:ID>
+                <cac:Attachment>
+                    <cbc:EmbeddedDocumentBinaryObject mimeCode="application/pdf" filename=%s>
+                        %s
+                    </cbc:EmbeddedDocumentBinaryObject>
+                </cac:Attachment>
+            </cac:AdditionalDocumentReference>
```

Comment:
Add this inside a new template and remove the duplicate inside `addons/account_edi_ubl/data/ubl_templates.xml`

Reply:
Forget it: Better to not force ppl to upgrade the module